ressurshefte/ressurshefte.md

```diff
@@ -154,15 +154,14 @@ Hint: sjekk DCAT-AP-NO-spesifikasjonen for hvilke felt som er obligatoriske for
 @prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix dct:    <http://purl.org/dc/terms/> .
 @prefix dcat:   <http://www.w3.org/ns/dcat#> .
-
```

Comment:
Tok med denne for å tydeliggjøre mellom eksterne vokabularer/namespaces og det egendefinerte. Kan ev. legge til kommentar om det.